### PR TITLE
release: commit results of update-latest-release

### PR DESCRIPTION
**Pull request by HashidaTKS**
Change to commit results of `make update-latest-release`.

### CODE REVIEW

**Comment by HashidaTKS** [comment]
https://github.com/groonga/groonga/issues/1440

File: doc/source/install/almalinux.rst [hunk 1]
```diff
@@ -69,9 +69,9 @@ Install required packages to build Groonga::
 
 Download source::
 
-  % wget https://packages.groonga.org/source/groonga/groonga-12.1.0.tar.gz
-  % tar xvzf groonga-12.1.0.tar.gz
-  % cd groonga-12.1.0
+  % wget https://packages.groonga.org/source/groonga/groonga-12.1.1.tar.gz
+  % tar xvzf groonga-12.1.1.tar.gz
+  % cd groonga-12.1.1
```
Comment (komainu8) [inline doc/source/install/almalinux.rst:74]: Is this need?

File: doc/source/install/amazon_linux.rst [hunk 2]
```diff
@@ -49,9 +49,9 @@ Install required packages to build Groonga::
 
 Download source::
 
-  % wget https://packages.groonga.org/source/groonga/groonga-12.1.0.tar.gz
-  % tar xvzf groonga-12.1.0.tar.gz
-  % cd groonga-12.1.0
+  % wget https://packages.groonga.org/source/groonga/groonga-12.1.1.tar.gz
+  % tar xvzf groonga-12.1.1.tar.gz
+  % cd groonga-12.1.1
```
Comment (komainu8) [inline doc/source/install/amazon_linux.rst:54]: Is this need?

File: doc/source/install/centos.rst [hunk 3]
```diff
@@ -54,9 +54,9 @@ Install required packages to build Groonga::
 
 Download source::
 
-  % wget https://packages.groonga.org/source/groonga/groonga-12.1.0.tar.gz
-  % tar xvzf groonga-12.1.0.tar.gz
-  % cd groonga-12.1.0
+  % wget https://packages.groonga.org/source/groonga/groonga-12.1.1.tar.gz
+  % tar xvzf groonga-12.1.1.tar.gz
+  % cd groonga-12.1.1
```
Comment (komainu8) [inline doc/source/install/centos.rst:59]: Is this need?

File: doc/source/install/cmake.rst [hunk 4]
```diff
@@ -37,15 +37,15 @@ GNU/Linux or UNIX
 
 .. code-block:: console
 
-   $ wget https://packages.groonga.org/source/groonga/groonga-12.1.0.tar.gz
-   $ tar xvzf groonga-12.1.0.tar.gz
+   $ wget https://packages.groonga.org/source/groonga/groonga-12.1.1.tar.gz
+   $ tar xvzf groonga-12.1.1.tar.gz
```
Comment (komainu8) [inline doc/source/install/cmake.rst:41]: Is this need?

**Comment by HashidaTKS** [comment]
Ah, Sorry. test executing results are included...

**Comment by HashidaTKS** [comment]
@komainu8 

Thanks, I have reverted needless changes.

**Pull request merged**